package-lock.json

```diff
@@ -2123,7 +2123,7 @@
     "d64": {
       "version": "1.0.0",
       "resolved": "https://registry.npmjs.org/d64/-/d64-1.0.0.tgz",
-      "integrity": "sha1-QAKofoUMv8n52XBrYPymE6MzbpA="
+      "integrity": "sha512-5eNy3WZziVYnrogqgXhcdEmqcDB2IHurTqLcrgssJsfkMVCUoUaZpK6cJjxxvLV2dUm5SuJMNcYfVGoin9UIRw=="
```

Comment:
How did these get it again :disappointed: 

```
$ curl https://registry.npmjs.org/d64 -s | jq '.versions."1.0.0".dist'
{
  "shasum": "4002a87e850cbfc9f9d9706b60fca613a3336e90",
  "tarball": "https://registry.npmjs.org/d64/-/d64-1.0.0.tgz"
}
```
There are no sha152 integrity hashes available for this package.

Reply:
(╯°□°）╯︵ ┻━┻